frontend/src/components/common/authorized-menu/styles.module.scss

```diff
@@ -1,7 +1,10 @@
 .menu {
+  position: sticky;
+  top: 96px;
   display: flex;
   flex-direction: column;
   min-width: 270px;
+  height: 89.6vh;
```

Comment:
is it okay to do it through 'vh' ?

Reply:
It must have fixed height, in other case it will stay relative